### ISSUE TITLE
Install fails on some system

### ISSUE THREAD

**Issue by jorol** [gbv/Catmandu-PICA#34]
see http://matrix.cpantesters.org/?dist=Catmandu-PICA+0.16.

Error:
```terminal

#   Failed test 'use Catmandu::Importer::PICA;'
#   at t/00-load.t line 6.
#     Tried to use 'Catmandu::Importer::PICA'.
#     Error:  Can't locate PICA/Parser/XML.pm in @INC (@INC contains: /home/stro/cpan/build/5.12.4/build/Catmandu-PICA-0.16-8qUGDg/blib/lib /home/stro/cpan/build/5.12.4/build/Catmandu-PICA-0.16-8qUGDg/blib/arch /home/stro/perl/5.12.4/lib/site_perl/5.12.4/x86_64-linux /home/stro/perl/5.12.4/lib/site_perl/5.12.4 /home/stro/perl/5.12.4/lib/5.12.4/x86_64-linux /home/stro/perl/5.12.4/lib/5.12.4 .) at /home/stro/cpan/build/5.12.4/build/Catmandu-PICA-0.16-8qUGDg/blib/lib/Catmandu/Importer/PICA.pm line 6, <DATA> line 4.
# BEGIN failed--compilation aborted at /home/stro/cpan/build/5.12.4/build/Catmandu-PICA-0.16-8qUGDg/blib/lib/Catmandu/Importer/PICA.pm line 6, <DATA> line 4.
# Compilation failed in require at t/00-load.t line 6, <DATA> line 4.
# BEGIN failed--compilation aborted at t/00-load.t line 6, <DATA> line 4.

#   Failed test 'use Catmandu::Exporter::PICA;'
#   at t/00-load.t line 6.
#     Tried to use 'Catmandu::Exporter::PICA'.
#     Error:  Can't locate PICA/Writer/Plus.pm in @INC (@INC contains: /home/stro/cpan/build/5.12.4/build/Catmandu-PICA-0.16-8qUGDg/blib/lib /home/stro/cpan/build/5.12.4/build/Catmandu-PICA-0.16-8qUGDg/blib/arch /home/stro/perl/5.12.4/lib/site_perl/5.12.4/x86_64-linux /home/stro/perl/5.12.4/lib/site_perl/5.12.4 /home/stro/perl/5.12.4/lib/5.12.4/x86_64-linux /home/stro/perl/5.12.4/lib/5.12.4 .) at /home/stro/cpan/build/5.12.4/build/Catmandu-PICA-0.16-8qUGDg/blib/lib/Catmandu/Exporter/PICA.pm line 4, <DATA> line 4.
# BEGIN failed--compilation aborted at /home/stro/cpan/build/5.12.4/build/Catmandu-PICA-0.16-8qUGDg/blib/lib/Catmandu/Exporter/PICA.pm line 4, <DATA> line 4.
# Compilation failed in require at t/00-load.t line 6, <DATA> line 4.
# BEGIN failed--compilation aborted at t/00-load.t line 6, <DATA> line 4.

#   Failed test 'use Catmandu::Fix::pica_map;'
#   at t/00-load.t line 6.
#     Tried to use 'Catmandu::Fix::pica_map'.
#     Error:  Can't locate PICA/Path.pm in @INC (@INC contains: /home/stro/cpan/build/5.12.4/build/Catmandu-PICA-0.16-8qUGDg/blib/lib /home/stro/cpan/build/5.12.4/build/Catmandu-PICA-0.16-8qUGDg/blib/arch /home/stro/perl/5.12.4/lib/site_perl/5.12.4/x86_64-linux /home/stro/perl/5.12.4/lib/site_perl/5.12.4 /home/stro/perl/5.12.4/lib/5.12.4/x86_64-linux /home/stro/perl/5.12.4/lib/5.12.4 .) at /home/stro/cpan/build/5.12.4/build/Catmandu-PICA-0.16-8qUGDg/blib/lib/Catmandu/Fix/pica_map.pm line 9, <DATA> line 4.
# BEGIN failed--compilation aborted at /home/stro/cpan/build/5.12.4/build/Catmandu-PICA-0.16-8qUGDg/blib/lib/Catmandu/Fix/pica_map.pm line 9, <DATA> line 4.
# Compilation failed in require at t/00-load.t line 6, <DATA> line 4.
# BEGIN failed--compilation aborted at t/00-load.t line 6, <DATA> line 4.
# Testing Catmandu::PICA 0.16, Perl 5.012004, /home/stro/perl/5.12.4/bin/perl
# Looks like you failed 3 tests of 4.
```

PICA::Data is not installed although it´s defined as prerequisite in cpanfile and META.json.

**Comment by jorol** [comment]
I think the old Makefile.PL causes the problem. Should we delete or update it?

**Comment by nichtich** [comment]
`milla build` creates `Build.PL` so `Makefile.PL` is obsolete.

**Comment by jorol** [comment]
solved 10ba5be5bb751f1b8b585fe7da5d7f56e8a5d218
